pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -4540,3 +4541,135 @@ func TestDisallowsInvalidFormatOptions(t *testing.T) {
 		}
 	}
 }
+
+type importSafeVisitor struct {
+	ctx    *tree.EvalContext
+	isSafe bool
+}
+
+func (v *importSafeVisitor) VisitPre(expr tree.Expr) (bool, tree.Expr) {
+	return v.isSafe, expr
+}
+
+var importSafeFunctions = map[string]struct{}{
+	"current_timestamp": {},
+	"now":               {},
+	"localtimestamp":    {},
+}
+
+func (v *importSafeVisitor) VisitPost(expr tree.Expr) (newNode tree.Expr) {
+	if fn, ok := expr.(*tree.FuncExpr); ok {
+		_, v.isSafe = importSafeFunctions[fn.Func.String()]
```

Comment:
Here you could modify the funcExpr to be the modified function that you want I think.

Reply:
Yeah -- i was doing that -- that's where I was using evalctx... probably not doing correctly though.

Regardless, since I only look at functions, and not say on casts, etc, I didn't think it's worth it.
Since we will eval these expressions only once, I decided that gain is too little to worry about it.